src/opnsense/service/conf/actions.d/actions_gateways.conf

```diff
@@ -0,0 +1,26 @@
+[start]
```

Comment:
is all this boilerplate really necessary? you also can't start a specific instance here (parameters:%s missing)

Reply:
It's not pretty, i agree.. But for us to hook into `updateServiceControlUI();`, the assumption of `<service> <start/stop/restart/status>` is necessary. The generic `actions_service.conf` assumes the reverse: `<start/stop/restart/status> <service>`.

This would also make sense from the perspective of modularization, though if we can make this more generic in general we also need to make sure other modules follow the same pattern.

Reply:
ok let's try to address this at a different time

Reply:
Can probably go, when removing the service controller and opt for an "apply" action only